source/includes/day-trip-api.md

```diff
@@ -0,0 +1,869 @@
+# Day Trip API
+
+Day trips combine comfortable door‑to‑door transportation with guided sightseeing. Each product is a pre‑planned excursion with a fixed itinerary. Customers can discover available day trips by country or by geographic coordinates, inspect the full itinerary, and then book the trip for a specific departure time. Unlike the point‑to‑point trips from the core API, Day Trip products cannot be customised with additional stops or different routes – the itinerary and sightseeing stops are predefined.
+
+The Day Trips API allows partners to search, browse, and book curated day trip packages.
+
+## Authentication
```

Comment:
I think, we should have just one common "Authentication" section, not two.

Reply:
removed

Reply:
done: c7d392a8d8088cb19ddbe8b27b24ad8b6404d947